z3/src/z3_solver.cpp

```diff
@@ -7,6 +7,9 @@
 
 #include <iostream>
 
+// TEMP for conversions
```

Comment:
why is this temp and what will it be replaced after that? I would explain in the comment maybe.

Reply:
Good idea, addressed with 6ead7ccfadad9877af9ff457dfab4dd6c4676fa9.